src/UI/SlotArea.cpp

```diff
@@ -1114,6 +1116,7 @@ void cSlotAreaAnvil::UpdateResult(cPlayer & a_Player)
 				Output.Empty();
 				SetSlot(2, a_Player, Output);
 				m_ParentWindow.SetProperty(0, 0, a_Player);
+				m_ParentWindow.SendSlot(a_Player, this, 2);
```

Comment:
I was having some trouble with the output slot updating correctly in the client when I commented the .SendSlot calls out, I'll look more into it and try to figure out why SetSlot isn't sending the slot, or why my client isn't picking up on it (to reproduce: just stick armor and a Protection enchanted book in an anvil, observe the output not appear. I'm using a 1.12 client).

If you want I can open a separate issue for this.